scripts/main.js

```diff
@@ -0,0 +1,21 @@
+
+// scroll top button
+
+$(function(){
```

Comment:
fajnie by bylo napisac to samemu w czystym JS, zamiast dodawac jQuery :).

Ale moze zostac, gdyz BS4 uzywa rowniez jQuery :).